pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -291,6 +295,20 @@ def test_task_progress_report(self):
                     0
                 )
 
+    def test_unit_count_on_repo(self):
+        """
+        Verify that the sync added the correct number of units to the repo.
```

Comment:
Flake8 is asking for a line to be inserted between the summary line ("Verify that the sync added...") and the remainder of the docstring. This will pass:

```
"""
Verify that the sync added the correct number of units to the repo.

Looks at the content counts on the repo.
"""
```

This will pass too (it's the docstring form used elsewhere in the project):

```
"""Verify that the sync added the correct number of units to the repo.

Looks at the content counts on the repo.

"""
```